src/uwtools/cli.py

```diff
@@ -145,17 +145,19 @@ def _add_subparser_config_realize(subparsers: Subparsers) -> ActionChecks:
     optional = _basic_setup(parser)
     _add_arg_input_file(optional)
     _add_arg_input_format(optional, choices=FORMATS)
+    _add_arg_update_file(optional)
```

Comment:
In other sections this is called `update_config` should we keep this consistent across files?

Reply:
I don't think so: This is the CLI, so users can only provide _files_ -- they cannot provide Python `dict` objects or `Config` objects on the command line. This is why it's also e.g. `_add_arg_input_file()` just above. Once we're at the API and beyond, there's a chance that users may be providing in-memory `dict` or `Config` objects, so the terminology changes from "file" to "config" to reflect the ambiguity.